tests/unit-tests/blocks/test-class-sensei-course-list-filter-block.php

```diff
@@ -254,7 +256,14 @@ public function testCourseFilterBlock_WhenFilteredForCompletedCourses_RendersThe
 		$this->login_as( $student );
 		$this->manuallyEnrolStudentInCourse( $student, $this->course1->ID );
 		$this->manuallyEnrolStudentInCourse( $student, $this->course2->ID );
-		Sensei_Utils::update_course_status( $student, $this->course2->ID, 'complete' );
+		$this->prevent_wp_redirect();
+
+		try {
+			Sensei_Utils::update_course_status( $student, $this->course2->ID, 'complete' );
+		} catch ( Sensei_WP_Redirect_Exception $e ) {
+			assert( true );
+		}
```

Comment:
I'm guessing this is a lovely side effect related to having the pages now. 😄

Reply:
Yeah 😄 Previously it had nowhere to go to after lesson completions 😄